.github/workflows/build-from-fork.yaml

```diff
@@ -60,7 +60,11 @@ jobs:
           fi
 
           # If environment not active, activate it
-          if ! $(platform env --format plain --columns title,status | grep '${{ env.BRANCH_TITLE }}' | grep -q Active); then
-            echo "Activating environment"
+          if ! $(platform environment --format plain --columns title,status | grep '${{ env.BRANCH_TITLE }}' | grep -q Active); then
+            echo "::notice::Updating Platform.sh environment title for better tracking..."
+            # in this case we DO want to staticaly include `pr-##` since we're referring to a pull request number
+            newTitle="(pr-${{ github.event.number }}) ${{ github.event.pull_request.title }}"
+            platform environment:info title "${newTitle}" -e ${{ env.BRANCH_TITLE }}
+            echo "::notice::Activating environment ${{ env.BRANCH_TITLE }}"
```

Comment:
@gilzow Do we want to do this same thing here so everything is the same?

https://github.com/platformsh/platformsh-docs/blob/5c46a76de852959699df096977543e164c01774d/.github/workflows/manage-environment.yaml#L16-L35

Reply:
cc @AnouckColson - thoughts on your side?

Reply:
@gilzow @chadwcarlson 👍 I'm all in favour of consistency.

Reply:
Added. For now I added it in the other workflow, but at some point I want to re-evaluate these workflows to see if we can make them more streamlined.